hw/ip/prim/lint/prim_ram_1r1w_async_adv.waiver

```diff
@@ -0,0 +1,14 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# waiver file for prim_ram_1r1w_async_adv
+
+waive -rules {HIER_BRANCH_NOT_READ} -location {prim_ram_1r1w_async_adv.sv} -regexp {Net 'rst_a_ni' is not read from} \
+      -comment "rst_a_ni is only used in assertions in certain configurations."
+
+waive -rules {INPUT_NOT_READ} -location {prim_ram_1r1w_async_adv.sv} -regexp {Input port 'rst_a_ni' is not read from, instance} \
+      -comment "rst_a_ni is only used in assertions in certain configurations."
+
+waive -rules {RESET_USE} -location {prim_ram_1r1w_async_adv.sv} -regexp {'rst_b_ni' is connected to 'prim_ram_1r1w' port 'rst_b_ni', and used as an asynchronous reset} \
+      -comment "rst_b_ni is a legal asynchronous reset."
```

Comment:
A this is now the `rst_b_ni` port of the `prim_ram_1r1w_async_adv.sv`. The `rst_a_ni` waiver is in a different commit and I think in a different waiver file. Can you try putting them into the same file and maybe make the comment more precise (see my other review comment please)?